--- Lmod-7.4.3/init/bash.in	2017-04-03 19:04:43.932891093 +0200
+++ Lmod-7.4.3-new/init/bash.in	2017-04-03 19:06:46.832137320 +0200
@@ -1,4 +1,6 @@
 # -*- shell-script -*-
+xtrace_saved="$(set -o | grep xtrace | grep 'on')";
+set +x
 #
 ########################################################################
 # Start Lmod BASHENV
@@ -144,6 +146,9 @@
 ########################################################################
 # End Lmod BASHENV
 ########################################################################
+[ -n "$xtrace_saved" ] && set -x 
+unset xtrace_saved
+
 #
 # Local Variables:
 # mode: shell-script
