diff -ruN modules-3.1.6.orig/doc/Makefile.in modules-3.1.6/doc/Makefile.in
--- modules-3.1.6.orig/doc/Makefile.in	2000-06-28 02:17:33.000000000 +0200
+++ modules-3.1.6/doc/Makefile.in	2005-11-02 13:52:40.706623696 +0100
@@ -34,13 +34,13 @@
 
 MAN_FILES= module.1 modulefile.4
 
-MAKEDIRS=  $(man1dir) $(man4dir)
+MAKEDIRS=  $(DESTDIR)$(man1dir) $(DESTDIR)$(man4dir)
 
 all:
 
 install: all install-dirs
-	$(INSTALL_DATA) module.1     $(man1dir)
-	$(INSTALL_DATA) modulefile.4 $(man4dir)
+	$(INSTALL_DATA) module.1     $(DESTDIR)$(man1dir)
+	$(INSTALL_DATA) modulefile.4 $(DESTDIR)$(man4dir)
 
 install-dirs:
 	@for i in $(MAKEDIRS) ; do \
diff -ruN modules-3.1.6.orig/etc/Makefile.in modules-3.1.6/etc/Makefile.in
--- modules-3.1.6.orig/etc/Makefile.in	2001-07-09 20:21:37.000000000 +0200
+++ modules-3.1.6/etc/Makefile.in	2005-11-02 13:52:44.783003992 +0100
@@ -32,13 +32,13 @@
 
 BINFILES= add.modules mkroot
 
-MAKEDIRS= $(bindir)
+MAKEDIRS= $(DESTDIR)$(bindir)
 
 all:    $(BINFILES)
 
 install: all install-dirs
-	$(INSTALL_PROGRAM) add.modules $(bindir)
-	$(INSTALL_PROGRAM) mkroot $(bindir)
+	$(INSTALL_PROGRAM) add.modules $(DESTDIR)$(bindir)
+	$(INSTALL_PROGRAM) mkroot $(DESTDIR)$(bindir)
 
 install-dirs:
 	@for i in $(MAKEDIRS) ; do \
diff -ruN modules-3.1.6.orig/ext/Makefile.in modules-3.1.6/ext/Makefile.in
--- modules-3.1.6.orig/ext/Makefile.in	2000-06-28 02:17:32.000000000 +0200
+++ modules-3.1.6/ext/Makefile.in	2005-11-02 13:52:49.970215416 +0100
@@ -32,12 +32,12 @@
 
 BINFILES= add.ext
 
-MAKEDIRS= $(bindir)
+MAKEDIRS= $(DESTDIR)$(bindir)
 
 all:    $(BINFILES)
 
 install: all install-dirs
-	$(INSTALL_PROGRAM) add.ext $(bindir)
+	$(INSTALL_PROGRAM) add.ext $(DESTDIR)$(bindir)
 
 install-dirs:
 	@for i in $(MAKEDIRS) ; do \
diff -ruN modules-3.1.6.orig/init/Makefile.in modules-3.1.6/init/Makefile.in
--- modules-3.1.6.orig/init/Makefile.in	2001-01-17 00:24:09.000000000 +0100
+++ modules-3.1.6/init/Makefile.in	2005-11-02 13:52:53.993603768 +0100
@@ -33,20 +33,20 @@
 
 INIT_FILES= bash csh ksh perl python sh tcsh zsh ksh .modulespath
 
-MAKEDIRS= $(initdir)
+MAKEDIRS= $(DESTDIR)$(initdir)
 
 all:    $(INIT_FILES)
 
 install: all install-dirs
-	$(INSTALL_DATA) bash $(initdir)
-	$(INSTALL_DATA) csh  $(initdir)
-	$(INSTALL_DATA) perl $(initdir)
-	$(INSTALL_DATA) python $(initdir)
-	$(INSTALL_DATA) sh   $(initdir)
-	$(INSTALL_DATA) tcsh $(initdir)
-	$(INSTALL_DATA) zsh  $(initdir)
-	$(INSTALL_DATA) ksh  $(initdir)
-	$(INSTALL_DATA) .modulespath  $(initdir)
+	$(INSTALL_DATA) bash $(DESTDIR)$(initdir)
+	$(INSTALL_DATA) csh  $(DESTDIR)$(initdir)
+	$(INSTALL_DATA) perl $(DESTDIR)$(initdir)
+	$(INSTALL_DATA) python $(DESTDIR)$(initdir)
+	$(INSTALL_DATA) sh   $(DESTDIR)$(initdir)
+	$(INSTALL_DATA) tcsh $(DESTDIR)$(initdir)
+	$(INSTALL_DATA) zsh  $(DESTDIR)$(initdir)
+	$(INSTALL_DATA) ksh  $(DESTDIR)$(initdir)
+	$(INSTALL_DATA) .modulespath  $(DESTDIR)$(initdir)
 
 install-dirs:
 	@for i in $(MAKEDIRS) ; do \
diff -ruN modules-3.1.6.orig/Makefile.in modules-3.1.6/Makefile.in
--- modules-3.1.6.orig/Makefile.in	2001-11-05 22:50:41.000000000 +0100
+++ modules-3.1.6/Makefile.in	2005-11-02 13:52:27.404645904 +0100
@@ -72,7 +72,7 @@
 HDRS =	modules_def.h
 ACS =	configure Makefile.in config.h.in configure.in acconfig.h aclocal.m4 
 
-MAKEDIRS = $(bindir) $(filedir)
+MAKEDIRS = $(DESTDIR)$(bindir) $(DESTDIR)$(filedir)
 
 ## X_CFLAGS, X_LIBS from AC_PATH_XTRA
 ALL_CFLAGS = -I.. -I$(srcdir) @TCL_INC_DIR@ @TCLX_INC_DIR@ @X_CFLAGS@ $(CFLAGS)
@@ -115,7 +115,7 @@
 	./config.status --recheck
 
 install:	all install-dirs
-	$(INSTALL_PROGRAM) modulecmd $(bindir)/modulecmd
+	$(INSTALL_PROGRAM) modulecmd $(DESTDIR)$(bindir)/modulecmd
 	(cd init;         $(MAKE) install)
 	(cd doc;          $(MAKE) install)
 	(cd etc;          $(MAKE) install)
diff -ruN modules-3.1.6.orig/modulefiles/Makefile.in modules-3.1.6/modulefiles/Makefile.in
--- modules-3.1.6.orig/modulefiles/Makefile.in	2001-07-11 23:56:35.000000000 +0200
+++ modules-3.1.6/modulefiles/Makefile.in	2005-11-02 13:53:01.178511496 +0100
@@ -31,18 +31,18 @@
 
 MOD_FILES= modules module-info null use.own dot version
 
-MAKEDIRS= $(filedir) $(versdir)
+MAKEDIRS= $(DESTDIR)$(filedir) $(DESTDIR)$(versdir)
 
 all:    $(MOD_FILES)
 
 install: all install-dirs
-	$(INSTALL_DATA) modules $(filedir)
-	$(INSTALL_DATA) module-cvs $(filedir)
-	$(INSTALL_DATA) module-info $(filedir)
-	$(INSTALL_DATA) null  $(filedir)
-	$(INSTALL_DATA) use.own $(filedir)
-	$(INSTALL_DATA) dot $(filedir)
-	$(INSTALL_DATA) version $(versdir)/@VERSION@
+	$(INSTALL_DATA) modules $(DESTDIR)$(filedir)
+	$(INSTALL_DATA) module-cvs $(DESTDIR)$(filedir)
+	$(INSTALL_DATA) module-info $(DESTDIR)$(filedir)
+	$(INSTALL_DATA) null  $(DESTDIR)$(filedir)
+	$(INSTALL_DATA) use.own $(DESTDIR)$(filedir)
+	$(INSTALL_DATA) dot $(DESTDIR)$(filedir)
+	$(INSTALL_DATA) version $(DESTDIR)$(versdir)/@VERSION@
 
 install-dirs:
 	@for i in $(MAKEDIRS) ; do \
